src/main/java/mekanism/common/block/states/BlockStateMachine.java

```diff
@@ -367,7 +367,88 @@ public double getUsage() {
+                    storage = MekanismConfig.current().storage.energizedSmelterStorage.val();
+                    break;
+                case TELEPORTER:
+                    storage = MekanismConfig.current().storage.teleporterStorage.val();
+                    break;
+                case ELECTRIC_PUMP:
+                    storage = MekanismConfig.current().storage.electricPumpStorage.val();
+                    break;
+                case CHARGEPAD:
+                    storage = MekanismConfig.current().storage.chargePadStorage.val();
+                    break;
+                case ROTARY_CONDENSENTRATOR:
+                    storage = MekanismConfig.current().storage.rotaryCondensentratorStorage.val();
+                    break;
+                case CHEMICAL_OXIDIZER:
+                    storage = MekanismConfig.current().storage.oxidationChamberStorage.val();
+                    break;
+                case CHEMICAL_INFUSER:
+                    storage = MekanismConfig.current().storage.chemicalInfuserStorage.val();
+                    break;
+                case CHEMICAL_INJECTION_CHAMBER:
+                    storage = MekanismConfig.current().storage.chemicalInjectionChamberStorage.val();
+                    break;
+                case ELECTROLYTIC_SEPARATOR:
+                    storage = MekanismConfig.current().storage.electrolyticSeparatorStorage.val();
+                    break;
+                case PRECISION_SAWMILL:
+                    storage = MekanismConfig.current().storage.precisionSawmillStorage.val();
+                    break;
+                case CHEMICAL_DISSOLUTION_CHAMBER:
+                    storage = MekanismConfig.current().storage.chemicalDissolutionChamberStorage.val();
+                    break;
+                case CHEMICAL_WASHER:
+                    storage = MekanismConfig.current().storage.chemicalWasherStorage.val();
+                    break;
+                case CHEMICAL_CRYSTALLIZER:
+                    storage = MekanismConfig.current().storage.chemicalCrystallizerStorage.val();
+                    break;
+                case SEISMIC_VIBRATOR:
+                    storage = MekanismConfig.current().storage.seismicVibratorStorage.val();
+                    break;
+                case PRESSURIZED_REACTION_CHAMBER:
+                    storage = MekanismConfig.current().storage.pressurizedReactionBaseStorage.val();
+                    break;
+                case FLUIDIC_PLENISHER:
+                    storage = MekanismConfig.current().storage.fluidicPlenisherStorage.val();
+                    break;
+                case LASER:
+                    storage = MekanismConfig.current().storage.laserStorage.val();
+                    break;
+                case FORMULAIC_ASSEMBLICATOR:
+                    storage = MekanismConfig.current().storage.formulaicAssemblicatorStorage.val();
+                    break;
+                default:
```

Comment:
how does this go with factories?

Reply:
Not well, my bad.

Working on a fix now. The factory calculations as well as some of the GUI display elements have some hardwired constants.